addons/web/static/src/views/form/button_box/button_box.scss

```diff
@@ -105,15 +105,20 @@
         margin-left: -$border-width;
     }
 
-    .oe_stat_button, > div > button {
+    .oe_stat_button, 
+    div > .oe_stat_button, 
```

Comment:
I think that this selector is not needed. `.oe_stat_button` inside a div will still be targeted by the first selector